packages/cfd/src/Containers/__tests__/cfd-password-modal.spec.js

```diff
@@ -285,7 +285,7 @@ describe('<CFDPasswordModal/>', () => {
         expect(await screen.findByText('MockedMT5Icon')).toBeInTheDocument();
     });
 
-    it('should display IcDxtradeSyntheticPlatform icon in Success Dialog', async () => {
+    it('should display IcRebrandingDerivx icon in Success Dialog', async () => {
```

Comment:
I think having this test case twice does not make sense. You need to remove one of them since we removed the Financial icons from the project